### PR TITLE
Add mutations

### PR DESCRIPTION
**Pull request by alexbeno**

### CODE REVIEW

File: app/p/[slug]/contributions/[contributionId]/page.tsx [hunk 1]
```diff
@@ -19,27 +19,30 @@ function ContributionPage() {
...
       options: { enabled: !!project?.id },
     });
 
   const { data: projectMembersApplicationsData, hasNextPage: projectMembersApplicationsHasNextPage } =
     applicationsApiClient.queries.useInfiniteGetAllApplications({
       // queryParams: { projectId: project?.id, issueId: contributionId, isApplicantProjectMember: true },
       queryParams: { projectId: project?.id },
-      options: { enabled: !!project?.id },
+      // options: { enabled: !!project?.id },
+      options: { enabled: false },
```
Comment (NeoxAzrot) [inline app/p/[slug]/contributions/[contributionId]/page.tsx:45]: Delete this, use the old one

File: components/atoms/button/button.core.tsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ export function ButtonCore<C extends ElementType = "button">({
       data-loading={isLoading}
       data-disabled={isDisabled}
       className={cn(slots.base(), classNames?.base)}
-      onClick={onClick}
+      onClick={() => onClick?.()}
```
Comment (NeoxAzrot) [inline components/atoms/button/button.core.tsx:46]: Why is this necessary?

**Review by NeoxAzrot — CHANGES_REQUESTED** [review]

**Review by haydencleary — APPROVED** [review]
